src/controllers/controlpickermenu.cpp

```diff
@@ -266,13 +266,22 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
 
     // Loops
     QMenu* loopMenu = addSubmenu(tr("Looping"));
+    // add beatloop_activate and beatlooproll_activate to both the
+    // Loop and Beat-Loop menus to make sure users can find them.
+    QString beatloopActivateTitle = tr("Loop Selected Beats");
+    QString beatloopActivateDescription = tr("Create beat loop of selected beat size");
+    QString beatloopRollActivateTitle = tr("Loop Roll Selected Beats");
+    QString beatloopRollActivateDescription = tr("Create rolling beat loop of selected beat size");
+
     addDeckControl("loop_in", tr("Loop In"), tr("Loop In button"), loopMenu);
     addDeckControl("loop_out", tr("Loop Out"), tr("Loop Out button"), loopMenu);
     addDeckControl("loop_exit", tr("Loop Exit"), tr("Loop Exit button"), loopMenu);
-    addDeckControl("reloop_exit", tr("Reloop/Exit Loop"), tr("Reloop/Exit button"), loopMenu);
-    addDeckControl("loop_halve", tr("Loop Halve"), tr("Halve the current loop's length"), loopMenu);
-    addDeckControl("loop_double", tr("Loop Double"),
-                   tr("Double the current loop's length"), loopMenu);
+    addDeckControl("reloop_toggle", tr("Reloop/Exit Loop"), tr("Toggle loop on/off"), loopMenu);
+    addDeckControl("reloop_andstop", tr("Reloop And Stop"), tr("Enable loop, jump to Loop In point, and stop"), loopMenu);
```

Comment:
"Return" is not quite correct because the playhead may not have gotten to the loop in point yet. The loop in point could have been set previously, before the track was loaded to the deck.